### PR TITLE
CIF-1996: Add copyright headers to CSS and graphql files

### PR DESCRIPTION
**Pull request by laurentiumagureanu**
## Description

Added copyright headers to CSS and graphql files

## Related Issue

CIF-1996

## Motivation and Context

According to a comment on a completely different issue on OSS Advisory Board we should have copyright headers on CSS and GraphQL files as well.

## How Has This Been Tested?

Unit tests
Manually

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [x] I have added tests to cover my changes and the overall coverage did not decrease.
- [x] All unit tests pass on CircleCi.
- [x] I ran all tests locally and they pass.

### CODE REVIEW

**Comment by dplaton** [comment]
I remember we had some tooling that collected the queries made by the React components (@herzog31 knows more). I think this tooling wouldn't work with the change you did to the `.graphql` files.

File: react-components/src/queries/__test__/validateQueries.test.js [hunk 1]
```diff
@@ -33,7 +33,9 @@ describe.each([['2.4.0', magentoSchema240]])(
 
         it.each(files)('validates the GraphQL request from %s', file => {
             let query = fs.readFileSync(path.join(__dirname, '..', file), 'UTF-8'); // eslint-disable-line
```
Comment (cjelger) [inline react-components/src/queries/__test__/validateQueries.test.js:35]: You should rather simply dynamically `import` each query/file here as any normal exported js module value.

**Review by cjelger — COMMENTED** [review]
Related to @dplaton's comment, this will break https://github.com/adobe/commerce-cif-graphql-integration-reference/blob/master/src/documentation/generate.js#L69-L73 but it would also be simple to fix it there.

File: react-components/src/queries/__test__/validateQueries.test.js (hunk 1 above)
Comment (laurentiumagureanu) [reply react-components/src/queries/__test__/validateQueries.test.js:35]: tried that. didn't work for some reason.

Comment (laurentiumagureanu) [reply react-components/src/queries/__test__/validateQueries.test.js:35]: there seems to be a difference between the parsed query and the imported one:
parsed query:
```  
    {
        kind: 'Document',
        definitions: [
          {
            kind: 'OperationDefinition',
            operation: 'query',
            name: [Object],
            variableDefinitions: [],
            directives: [],
            selectionSet: [Object],
            loc: [Object]
          }
        ],
        loc: { start: 0, end: 146 }
     } 
```
imported query : 
```
    {
        default: {
          kind: 'Document',
          definitions: [ [Object] ],
          loc: { start: 0, end: 147 }
        }
    }
```

Comment (laurentiumagureanu) [reply react-components/src/queries/__test__/validateQueries.test.js:35]: Ignore my comments. I'll update the code.

**Comment by laurentiumagureanu** [comment]
@dplaton, @cjelger I opened a PR for the tool you mentioned 
https://github.com/adobe/commerce-cif-graphql-integration-reference/pull/40

**Comment by herzog31** [comment]
Did anyone ask for an exception? Changing all the graphql files including the webpack build and tests, just for the sake of adding copyright headers is ridiculous. Can't we simply put a license file in the folder where all queries are and say that it applies to all?

I'm fine with the changes to the css files.

**Pull request merged**